lib/hcl/commands.rb

```diff
@@ -111,9 +112,24 @@ def start *args
       if task.nil?
         fail "Unknown task alias, try one of the following: ", aliases.join(', ')
       end
+      note = args.join(' ')
```

Comment:
This is good functionality but it repeats a lot of code. You should consider creating a helper method of some kind that does most of the heavy lifting, and then your repeated pattern is just a method call.